docs/sources/panels-visualizations/visualizations/datagrid/index.md

```diff
@@ -0,0 +1,82 @@
+---
+aliases:
+  - ../../features/panels/datagrid_panel/
+  - ../../reference/datagrid/
+  - ../../visualizations/datagrid/
+description: Learn about datagrid panel visualization features.
+keywords:
+  - grafana
+  - dashboard
+  - panels
+  - datagrid
+  - datagrid panel
+  - datagrid options
+menuTitle: Datagrid
+title: Datagrid
+weight: 950
+---
+
+# Datagrid
+
+> **Note:** Grafana Datagrid panel is Experimental. This feature is supported by the engineering team on a best-effort basis, and breaking changes may occur without notice prior to general availability.
+
+The datagrid panel offers a user the ability to create or edit and fine-tune data within Grafana. As such, this panel can act as a data source for other panels
+inside a dashboard. Let us dive deeper into the flows and functionalities of this panel.
+
+![Datagrid panel](/media/docs/datagrid/screenshot-grafana-datagrid-panel.png)
+
+## Use cases
+
+The datagrid panel can be used to manipulate data queried from any data source or from a dragged and dropped file. It can then be used as a simple tabular
+visualization or the data can be modified and even removed altogether to create a blank slate. Any changes made to data source retrieved data trigger a confirmation modal that informs the user about the action they are about to take. Editing the dataset changes the data source to use the inbuilt `-- Grafana --` data source, thus losing the old data source settings and related queries, while also copying the current dataset into the dashboards model. After confirming the action, all data modifications are saved to the dashboard model.
```

Comment:
Is there documentation about the `-- Dashboard --` data source that you should link here?

Reply:
That is a good question, I have not found any docs related to the in-built `Dashboard` or `Grafana` datasources. @imatwawana do you know of any?